cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -47,12 +49,54 @@ func isPermanentError(err error) bool {
 }
 
 type driver struct {
-	sync.Mutex
 	client       coreclientset.Interface
 	pluginhelper *kubeletplugin.Helper
 	state        *DeviceState
 }
 
+// Protect the work in nodePrepareResource() and nodeUnprepareResource() under a
```

Comment:
comments on functions must start with the function name

Reply:
done